modules/ztp-deploying-a-site.adoc

```diff
@@ -182,6 +182,6 @@ The `SiteConfig` CR creates the following CRs on the hub cluster:
 * `ClusterDeployment`
 * `InfraEnv`
 * `NMStateConfig` - One per node
-* `ExtraManifestsConfigMap` - Extra manifests. The additional manifests include workload partitioning, chronyd, mountpoint hiding, sctp enablement, and more.
+* `ExtraManifestsConfigMap` - Extra manifests. The additional manifests include workload partitioning, mountpoint hiding, sctp enablement, and more. The extra manifests are automatically merged into a single manifest per each role, which is named as `predefined-extra-manifests`.
```

Comment:
The context of "per role" isn't clear here. Can we expand on that slightly? I *believe* it is per MachineConfigPool role but needs confirmation. @tliu2021 ?

Reply:
Yes, it is per MachineConfigPool role.

Reply:
@tliu2021 @imiller0 Updated. Could you please take another look and let me know if it looks good?